src/Runner.Worker/ActionManifestManager.cs

```diff
@@ -33,12 +33,13 @@ public interface IActionManifestManager : IRunnerService
     public sealed class ActionManifestManager : RunnerService, IActionManifestManager
     {
         private TemplateSchema _actionManifestSchema;
-        private IReadOnlyList<String> _fileTable;
-
+        private IHostContext _hostContext;
```

Comment:
you don't need this one.

HostContext is in RunnerService.

Reply:
TODO: we don't need to store _hostContext, we can just use HostContext